src/Enterspeed.Source.Sdk/Domain/Connection/Response.cs

```diff
@@ -10,5 +11,8 @@ public class Response
         public string Message { get; set; }
         public Exception Exception { get; set; }
         public bool Success { get; set; }
+        public Dictionary<string, string> Errors { get; set; }
+        public string ErrorCode { get; set; }
+        public string ResponseContent { get; set; }
```

Comment:
As you already is in the Response object I think you could name it `Content` instead of `ResponseContent`.